### DISCUSSION TITLE
Missing cookie values

### DISCUSSION THREAD

**Discussion by Grundhunden** [ptz0n/homebridge-verisure#155] [General]
Hi, I have a question regarding retrieval of the cookies.

I am running: homebridge-verisure v2.0.0

Your documentation states that the cookie section is made up of 3 separate cookies:
    _"vid=myTopSecretToken",
    "vs-access=myAccessToken",
    "vs-refresh=myRefreshToken"_

And those are the 3 cookies you get when running the npc homebridge-verisure command
But in my case it returns a token, instead of the 3 cookies.
I cannot seem to find anything on this in the documentation or discussions.

Could you point me in the right direction please?

_npx homebridge-verisure
✔ What is your login email? · xxxxx@xxxx.com
✔ What is your password? · ********************

 One-time code sent.

✔ What is your one-time code? · xxxxxx

 Your config is ready.

{
  "platform": "verisure",
  "name": "Verisure",
  "email": "xxxxx@xxxx.com",
  "token": "vid=********************"
}_

**Comment by MarcusCmr** [comment]
Hi, 
I have the exact same problem.
MFA is activated for the account that I´m using for this, I tried to change it to an admin account, but it did not help.
I´m using a Verisure vbox micro with Yale doorman.
It´s been working fine with the token only until some time ago, maybe a couple of weeks?
Not sure what has changed, It was running fine with ver 1.16 before, now I have updated to 2.0.0, same problem.

**Reply by MarcusCmr** [reply]
Tried again with ver 1.16.
I got this from the log:

2023-03-21 11:50:08Verisure Bridge starting
2023-03-21 11:50:12Verisure BridgeLoaded plugin 'homebridge-verisure'
2023-03-21 11:50:12Verisure BridgeLoading 1 platforms...
2023-03-21 11:50:12Verisure BridgeWARNINGunhandled rejection: Error: Request failed with status code 404

Earlier with ver 2.0.0
It looked like this in my log:

2023-03-21 10:49:26Verisure Bridge starting
2023-03-21 10:49:29Verisure BridgeLoaded plugin 'homebridge-verisure'
2023-03-21 10:49:29Verisure BridgeLoading 1 platforms...
2023-03-21 10:49:29Verisure BridgeVerisureERRORDEPRECATED: Property "token" in config. Please see README to get and configure "cookies".
2023-03-21 10:49:30Verisure BridgeVerisureERRORUnable to get installations. Please check configured credentials: Request failed with status code 401

**Comment by Grundhunden** [comment]
I get the same with v2.0.0
_[3/20/2023, 5:07:17 PM] [verisure] Unable to get installations. Please check configured credentials: Request failed with status code 401_

**Comment by ptz0n** [comment]
You seem to use an old version of `homebridge-verisure`. You should be on `>=2.0.0`.

```bash
npm list -g | grep verisure
```

**Reply by MarcusCmr** [reply]
Hi,
Thanks for the reply, I´m running hoobs, and it seems to be all up to date.

npm list -g
/usr/lib
├── corepack@0.15.1
└── npm@8.19.3

I reinstalled homebridge-verisure plugin ver 2.0.0.
It´s the latest available.
But I still get the same result from npx homebridge-verisure, Only token as a result.

I have been very happy with this plug-in, hope we can find a solution.
Thanks

**Reply by ptz0n** [reply]
Run `npx clear-npx-cache` and then try again.

**Reply by Grundhunden** [reply]
npx clear-npx-cache fixed the issue - thanks!!
